triphouse-project/styles/reviews-section.sass

```diff
@@ -0,0 +1,268 @@
+.reviews-section
+  padding: 0 104px 120px
+  text-align: center
+  position: relative
+
+.reviews-panel
+  padding-top: 104px
+  overflow: hidden
+
+.reviews-h2-hidden
+  display: none
+
+.review-item
+  text-align: left
+  display: flex
+  flex-direction: column
+  margin-left: 8px
+  margin-right: 8px
+  >a
+    height: 395px
+    position: relative
+    background-repeat: no-repeat
+    background-size: cover
+    background-position: bottom
+    &:hover::before
+      background-color: rgba(56, 56, 56, 0.4)
+      position: absolute
+      top: 88%
+      right: 0
+      bottom: 0
+      left: 0
+      padding: 16px 0 11px 24px
+      text-decoration: none
+      color: #FFFFFF
+
+.reviews-link-1
+  background: url("../assets/img/review-1.svg")
+  &:hover::before
+    content: "From 250 USD"
+
+.reviews-link-2
+  background: url("../assets/img/review-2.svg")
+  &:hover::before
+    content: "From 40 USD"
+
+.reviews-link-3
+  background: url("../assets/img/review-3.svg")
+  &:hover::before
+    content: "From 100 USD"
```

Comment:
is it necessary to add classname for each element? as far as I can see you can use `:nth-child`